### ISSUE TITLE
508 Compliance

### ISSUE THREAD

**Issue by ghost** [USGS-WiM/CyAN-Web#201]
Most of our features already have ARIA labels, but we need to make sure to add any that are missing, make use of other [ARIA Landmarks](https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/aria-landmarks/) and check the app for other accessibility issues. I'm going to try out the WAVE Chrome extension. 
More info: 
- https://www.wearediagram.com/blog/understanding-web-accessibility-and-section-508
- https://www.usgs.gov/survey-manual/6006-implementation-and-administration-section-508-rehabilitation-act
- https://www.section508.gov/test/
